### PR TITLE
Clarified the review instructions.

### PR DESCRIPTION
**Pull request by WilliamDenniss**

### CODE REVIEW

File: reviewing.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # How to review conformance results
 
-  1. Make sure the list of files matches [the one specified here](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr)
-  2. Look at `version.txt`.  Make sure the `client` and `server` versions match each other to the minor version level.  Make sure this also matches the `vX.Y` subdirectory that the PR is in.
-  3. Verify that the last line of `e2e.log` says "SUCCESS! -- `N` Passed | 0 Failed | 0 Pending | `M` Skipped PASS".  The exact value of `N` and `M` don't matter as long as we see `0 Failed | 0 Pending`.
+1. Verify that the list of files matches the
+[expected list](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr)
+
+2. Note the `vX.Y` subdirectory that the PR is in, this is the version of
+Kubernetes for which conformance being claimed, referenced as the "Conformance
```
Comment (dankohn) [inline reviewing.md]: ...conformance is being claimed...

**Comment by dankohn** [comment]
LGTM

@smarterclayton @timothysc Could you please confirm this matches your understanding of the process?

File: reviewing.md (hunk 1 above)
Comment (WilliamDenniss) [reply reviewing.md]: Thanks for the catch, fixed.

**Comment by smarterclayton** [comment]
LGTM

**Pull request merged**